src/shared/criterion-item/CriterionItem.tsx

```diff
@@ -9,12 +9,13 @@ import { FCProps } from 'src/shared/types/FCProps';
 interface OwnProps {
   title: string;
   prefix: string;
+  type: 'self' | 'peer';
```

Comment:
Let's not redefine this we can export rating props and use it's `RatingProps['type'] instead of redefine this

Reply:
in next pr will be done